src/main/resources/db/mysql/initDB.sql

```diff
@@ -59,3 +59,15 @@ CREATE TABLE IF NOT EXISTS visits (
   description VARCHAR(255),
   FOREIGN KEY (pet_id) REFERENCES pets(id)
 ) engine=InnoDB;
+
+/*Code added by Maria Carolina Avila for the APPT team*/
+CREATE TABLE IF NOT EXISTS vet_schedule (
+    vet_ID INT(4) NOT NULL PRIMARY KEY,
+    room_ID VARCHAR(5) NOT NULL,
+    day_available NUMERIC(1,0),
```

Comment:
Make this `NOT NULL` for consistency with H2, add a constraint
```suggestion
    day_available NUMERIC(1,0) NOT NULL CHECK (day_available BETWEEN 0 AND 6),
```